pkg/doc/signature/signer/signer.go

```diff
@@ -102,6 +104,7 @@ func (signer *DocumentSigner) signObject(context *Context, jsonLdObject map[stri
 		Domain:                  context.Domain,
 		Nonce:                   context.Nonce,
 		VerificationMethod:      context.VerificationMethod,
+		ProofPurpose:            defaultProofPurpose,
```

Comment:
There is also one more place where we call "NewProof()", shouldn't we use default there too?

Reply:
It's the place where we roughly speaking unmarshal `Proof`. So we should leave existent assignment:
```
ProofPurpose:            stringEntry(emap[jsonldProofPurpose]),
```